krping.c

```diff
@@ -1823,7 +1831,8 @@ static void krping_fr_test(struct krping_cb *cb)
 			fr.key = mr->rkey;
 			inv.ex.invalidate_rkey = mr->rkey;
 
-			size = prandom_u32() % cb->size;
+			// size = prandom_u32() % cb->size;
```

Comment:
Please remove the commented-out old code.  no need to keep it.  We have git to see the old code. :)